sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClient.java

```diff
@@ -400,7 +407,8 @@ public Response<BlobProperties> downloadToFileWithResponse(String filePath, Blob
     @Override
     public Response<BlobProperties> downloadToFileWithResponse(BlobDownloadToFileOptions options, Duration timeout,
         Context context) {
-        context = context == null ? Context.NONE : context;
+        context = context == null ? new Context("rangeAdjustment", adjustRangeThroughContext())
```

Comment:
For adding the adjusted length, we probably want to make the update in the `populateRequestConditionsAndContext` method similar to what we did in the async client.